packages/edit-site/src/components/sidebar-edit-mode/global-styles-sidebar.js

```diff
@@ -136,9 +136,12 @@ export default function GlobalStylesSidebar() {
 			closeLabel={ __( 'Close Styles' ) }
 			panelClassName="edit-site-global-styles-sidebar__panel"
 			header={
-				<Flex className="edit-site-global-styles-sidebar__header">
+				<Flex
+					className="edit-site-global-styles-sidebar__header"
+					gap={ 1 }
+				>
 					<FlexBlock style={ { minWidth: 'min-content' } }>
-						<h2 className="edit-site-global-styles-sidebar__header-title">
+						<h2 className="interface-complementary-area-header__title">
```

Comment:
I think the `.interface-complementary-area-header__title` CSS class is a CSS class that is expected to be used only inside the `ComplementaryArea` component. Therefore, I think it is better to avoid using this CSS class directly in another component. Is it possible to maintain the original CSS class and restore [this style](https://github.com/WordPress/gutenberg/pull/61331/files#diff-426840ed58101046fac53af274ef57ef4721323e285ae0337a341939a7d82e57L37-L39)?

Reply:
Sounds good, I added back via https://github.com/WordPress/gutenberg/pull/61331/commits/ba8fc0753503b8903973e6ab206c3ff993f1a0fe and https://github.com/WordPress/gutenberg/pull/61331/commits/e9b74007cb77a57c477cc17f58dcb2811011a2e0.